docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5196,6 +5196,11 @@ image](#create-an-image) section for more details.
                   - **Options** - key/value map of driver specific options.
         - **StopGracePeriod** – Amount of time to wait for the container to terminate before
           forcefully killing it.
+        - **DNSConfig** – Specification for DNS related configurations in
```

Comment:
Perhaps we should add it to the JSON example as well (just realized we didn't do that for the TTY option :smile:)